src/commands/staff/Find.ts

```diff
@@ -304,10 +303,10 @@ export class Find extends BaseCommand {
                         );
 
                     const time = new StringBuilder()
-                        .append(`Issued: \`${getDateTime(x.issuedAt)} GMT\``).appendLine()
+                        .append(`Issued: ${TimeUtilities.getDiscordTime({ time: x.issuedAt, style: "F" })}`).appendLine()
                         .append(
                             x.resolved
-                                ? `Resolved: \`${getDateTime(x.resolved.issuedAt)} GMT\``
+                                ? `Resolved: ${TimeUtilities.getDiscordTime({ time: x.resolved.issuedAt, style: "F" })}`
                                 : `Expires At: \`${expiresAtDisplay}\``
```

Comment:
And here

![image](https://user-images.githubusercontent.com/37031713/195476212-a8c6b753-e8f9-409d-837e-addc199c0a06.png)